op-devstack/sysgo/contracts.go

```diff
@@ -6,6 +6,12 @@ import (
 	"os"
 )
 
+type ContractPaths struct {
+	// must be absolute paths, without file:// prefix
+	FoundryArtifacts string
```

Comment:
Should we add an assertion for this somewhere then?

Reply:
The deployer code will already error if it's malformed, I think it's fine without assertion.